### PR TITLE
Update Pagers styleguidist documentation

### PR DESCRIPTION
**Pull request by captainsafia**

### CODE REVIEW

File: packages/presentational-components/src/components/pagers.md [hunk 1]
```diff
@@ -1,9 +1,15 @@
+The `Pagers` component is a container component that serves as the container for the pagers of a cell. Pagers are generally used to display information that is not an output. For example, you might use a pager to display the output of a cell magic. You'll generally have both a `Pagers` component and an `Outputs` component as part of your notebook cell so that you can display both pagers and outputs.
```
Comment (rgbkrk) [inline packages/presentational-components/src/components/pagers.md]: Classically, `Pagers` hold the introspection output (`?` and `??`).

```python
import pandas as pd

# in outputs
display(pd)

# in pager
pd?
```

screen shot 2018-11-01 at 10 14 10 am

Comment (willingc) [reply packages/presentational-components/src/components/pagers.md]: Some additional suggestions:

As a container component, `Pagers` manage a collection of the cell's pagers.  A pager is used to provide, but not modify, additional information. Common uses include introspection with `?` and `??` and help text for commands. For example, you might use a pager to display the output of a cell magic. 

A cell may have an `Outputs` component and a `Pagers` component. `Outputs` typically contain calculation results or actions on a cell, and `Pagers` hold information, like help text, found from introspecting a cell.

File: packages/presentational-components/src/components/pagers.md [hunk 2]
```diff
@@ -1,9 +1,15 @@
...
+
 ```jsx static
 import { Pagers } from "@nteract/presentational-components"
 ```
 
-Display a pager area
+Similar to other components, the `Pagers`  component provides a `hidden` flag that determines whether or not the contents of the component should be rendered. You can try it out below by toggling the value of the `hidden` prop in the code from `false` to `true`.
```
Comment (willingc) [inline packages/presentational-components/src/components/pagers.md:12]: Looks like an extra space between Pagers and component. 

...that determines if a Pager component should be rendered.

**Review by willingc — COMMENTED** [review]
Looks good Safia. I made some suggestions. Not sure if they are helpful but tossing out as a different way to define a difficult to describe term.

**Comment by captainsafia** [comment]
Thank you for the feedback everyone! I've made changes based on it.

**Comment by rgbkrk** [comment]
Awesome, thanks Safia!

**Pull request merged**